bugwarrior/services/gitlab.py

```diff
@@ -332,6 +332,7 @@ def _fetch(self, tmpl, **kwargs):
         if not self.verify_ssl:
             requests.packages.urllib3.disable_warnings()
         response = requests.get(url, headers=headers, verify=self.verify_ssl, **kwargs)
+        response.raise_for_status()
```

Comment:
Is there any advantage to this approach over catching the `IOError` that would be raised by `json_response` (https://github.com/ralphbean/bugwarrior/blob/develop/bugwarrior/services/__init__.py#L489)? If you prefer this approach, could you add a comment to this line like `# raise HTTPError`? Otherwise it wont be obvious what's going on here without browsing commit history.